opengapps-packages.mk

```diff
@@ -1,3 +1,4 @@
+include vendor/google/build/config.mk
```

Comment:
I can't see why this is needed, this should get included by the existing AOSP build system, even on M: https://android.googlesource.com/platform/build/+/android-6.0.0_r26/core/main.mk#106